application-task-test/application-task-test-docker/src/test/it/com/xwiki/task/test/ui/GanttIT.java

```diff
@@ -0,0 +1,318 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.task.test.ui;
+
+import java.util.List;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Order;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.xwiki.test.docker.junit5.UITest;
+import org.xwiki.test.ui.TestUtils;
+import org.xwiki.test.ui.po.CreatePagePage;
+
+import com.xwiki.task.model.Task;
+
+import org.xwiki.contrib.application.task.test.po.TaskManagerAdminConfigurationPage;
+import org.xwiki.contrib.application.task.test.po.TaskManagerGanttMacro;
+import org.xwiki.contrib.application.task.test.po.TaskManagerHomePage;
+import org.xwiki.contrib.application.task.test.po.TaskManagerInlinePage;
+import org.xwiki.contrib.application.task.test.po.TaskManagerViewPage;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.model.reference.EntityReference;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * UI Tests for the Gantt diagram macro. The application uses the frappe-gantt library to draw the diagram, with several
+ * custom patches. Updating the version of frappe-gantt might cause the tests to break.
+ *
+ * @version $Id$
+ * @since 3.7
```

Comment:
This would probably be in `3.8`, since we already have a `3.7` scheduled (the RC was already released)
Also, if you have the `since` added here, you don't need on each method. All methods have by default the same `since` as the class, so only methods that aren't since 3.8 will need a `since`